finance/bitcoin/Portfile

```diff
@@ -112,8 +112,12 @@ variant gui description {Build the Qt5 GUI} {
 }
 
 variant wallet description {Build with support for wallet} {
-    depends_lib-append port:sqlite3
+    depends_lib-append port:db48 \
+                       port:sqlite3
     configure.args-replace --enable-wallet=no --enable-wallet=yes
+    configure.env-append \
+        BDB_CFLAGS=-I${prefix}/include/db48 \
+        BDB_LIBS="-L${prefix}/lib/db48 -ldb_cxx-4.8"
```

Comment:
This is still quoted incorrectly, see my previous comment.

Reply:
This is the part in the log that tells you there was a problem with the env option, BTW:
```
DEBUG: parse_environment: skipping invalid entry: '-ldb_cxx-4.8"'
```

Reply:
Sorry, done this properly now.